CsCore/xUnitTests/xUnitTests.csproj

```diff
@@ -25,6 +25,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="StbImageWriteSharp" Version="1.16.7" />
```

Comment:
Note to myself, this lib I wanted to include for a while, there needs to be done some addition changes to include it in a way that it will also work in Unity but I can do this after merging your PR, so nothing to do for you